src/UI/Checkbox.elm

```diff
@@ -0,0 +1,46 @@
+module UI.Checkbox exposing (checkbox, radioButton)
+
+{-|
+
+
+# Atomic Elements
+
+@docs checkbox, radioButton
+
+**NOTE**:
+We have plans for "checklists" and "radioGroups", but this won't be created until usecase appears.
```

Comment:
So, I think we cannot have radio buttons without having the radiogroup behavior 🤔  not having the radio group now means that every time we need to use radio buttons we will need to implement the radiogroup behavior as checkboxes can show up alone, but this does not apply to radio buttons.